certified-connectors/Dynamics 365 Translation Service/apiDefinition.swagger.json

```diff
@@ -0,0 +1,887 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Dynamics Translation Service",
+    "version": "1.0.0",
+    "description": "Translation, Alignment, and Regeneration operations powered by Dynamics Translation Service.",
+    "contact": {
+      "name": "Dynamics Translation Service",
+      "url": "https://lcs.dynamics.com",
+      "email": "dtssup@microsoft.com"
+    }
+  },
+  "host": "lcsapi.lcs.dynamics.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "multipart/form-data",
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/dts/align/submit": {
+      "post": {
+        "consumes": [
+          "multipart/form-data"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "description": "Select the product type.",
+            "x-ms-summary": "Product Type",
+            "in": "formData",
+            "name": "productType",
+            "type": "string",
+            "x-ms-dynamic-values": {
+              "operationId": "Products",
+              "value-collection": "products",
+              "value-path": "id",
+              "value-title": "productName"
+            }
+          },
+          {
+            "description": "Select the product version.",
+            "x-ms-summary": "Product Version",
+            "in": "formData",
+            "name": "productVersion",
+            "type": "string",
+            "x-ms-dynamic-values": {
+              "operationId": "Product_Versions",
```

Comment:
@joshftb operationId properties should use pascal case for values. Please refer to [coding standards documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#actions-and-triggers).

Reply:
Done!